includes/KSES.php

```diff
@@ -47,13 +48,24 @@ public function register() {
 			return;
 		}
 
-		if ( ! current_user_can( 'unfiltered_html' ) ) {
-			add_filter( 'safe_style_css', [ $this, 'filter_safe_style_css' ] );
-			add_filter( 'wp_kses_allowed_html', [ $this, 'filter_kses_allowed_html' ], 10, 2 );
-			add_filter( 'content_save_pre', [ $this, 'filter_content_save_pre_before_kses' ], 0 );
-			add_filter( 'content_save_pre', [ $this, 'filter_content_save_pre_after_kses' ], 20 );
-			remove_filter( 'content_filtered_save_pre', 'wp_filter_post_kses' );
+		if ( current_user_can( 'unfiltered_html' ) ) {
+			return;
+		}
+
+		// The third param of the below filter was only added in WP 5.4.1.
+		if ( version_compare( get_bloginfo( 'version' ), '5.4.1', '>=' ) ) {
```

Comment:
As we also have some problems with 5.3 and gutenberg. It is worth having the conversion around making 5.4.1 the min version?

Reply:
I think so, yeah. Last time I checked, most users are on 5.6+ anyway, so data would back this up too.

Reply:
Created #7551, will discuss with product team